modules/qna/src/backend/setup.ts

```diff
@@ -45,6 +45,26 @@ export const initModule = async (bp: typeof sdk, botScopedStorage: Map<string, Q
   const buildSuggestions = async (event, question, confidence, intent, renderer): Promise<sdk.IO.Suggestion> => {
     const payloads = []
 
+    const config = await bp.config.getModuleConfigForBot('qna', event.botId)
+
+    if (config.saveQuestionData) {
```

Comment:
good idea, can you send this as another PR as I feel it's not related to your module proposal.

Reply:
With this we can save in the registry books how many times a question is being made, later while asking the customer if he find the answer useful, we can also save that in the module, etc etc..